mypage/index.md

```diff
@@ -0,0 +1,98 @@
+---
+layout: layouts/base.njk
+title: My Page
+---
+
+# How to add a page to the site
+
+### 1. Create a [https://www.markdowntutorial.com/](Markdown) file with an ".md" file extension.
```

Comment:
@cmneal26 @twalker 

What made you decide on line 8 to use an 'H3', and not an 'H2'?

 Claire, in this context, we focus on the task. It's either true that it was intentional or not, and this doesn't matter. What matters is whether there's anything interesting about this, or if it's a thing. The focus is on that line 8, because that's what this comment is about.

Reply:
A helpful attitude: what information can I add to a thread or PR to achieve my goals while being respectful of the goals of the other collaborators?

A sometimes unhelpful attitude: scared of ruining things, and making the wrong comment. This happens to me a lot. That's why I need to practice screen-share. Always think: "Given what I know about the situation, and a principle of transparency (which we defined in our core SCRUM team values) how can we carry out the current task.

An attitude unhelpful for task-completion: (1) situation is such, that a failure occurs, (2) people are stressed, and point fingers, (3) the task suffers, the work suffers, and people aren't happy.

Reply:
@cmneal26 

When I use `@cmneal26` it doesn't mean any action is required. If I use this, and don't ask you a question/suggestion, this means I just wanted to bring that thing to your notice. Unless you want to, there's usually no reason for you to attend to a comment from the team immediately.

Although we're not using priority tags/rankings, this is something that can help keep track of when/which tasks to work on in a given sprint. So, this is something that may very well be available, if you work in an agile environment as a tech writer at a tech company. And it could make your life easier. But keep in mind, it might seem a little weird/pressure.

Reply:
@cmneal26 

Again, no immediate priority. When you see this go ahead and follow these steps. This is my last comment for the day. I'll check in with you at our meeting in Friday.

- Read over Tim's Markdown document.
- Find something that you don't understand in the document, and ask a question as a GitHub Comment.
- Keep in Mind This Isn't a regular PR; it's an EXAMPLE one you can use as a reference. If a ticket assigned to you, and it involves some code, it might very well have an EXAMPLE PR linked somewhere in the ticket description.

Reply:
Oh, one last thing. I like this cool icon:

![cool-icon](https://user-images.githubusercontent.com/32662279/176543992-21291b5f-f907-4be6-b72e-4834f80acc2a.PNG)


See how there is a task icon and task count in the image PR. For example, if you're not sure why that PR is displaying a task icon, and some of the others didn't, you can ask Tim.